packages/SwingSet/src/kernel/dynamicVat.js

```diff
@@ -123,6 +129,7 @@ export function makeDynamicVatCreator(stuff) {
         enableSetup,
         enableInternalMetering: false,
         notifyTermination: metered ? notifyTermination : undefined,
+        notifyTerminationOnFailure,
```

Comment:
Confused by the asymmetry in default values here.

Reply:
@warner  why does `notifiyTermination` take undefined when metering is not present? What should `notifiyTerminationOnFailure` do?